### PR TITLE
Update Theme Preview Items

### PR DESCRIPTION
**Pull request by CrepeTF**
Updated the look of the theme previews

### Images
| Before | After |
| ------- | ------- |
| Before buying Korone NFTs | After buying Korone NFTs |

### CODE REVIEW

File: app/src/main/res/layout/pref_theme_item.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="118dp"
+    android:layout_width="114dp"
```
Comment (arkon) [inline app/src/main/res/layout/pref_theme_item.xml:5]: You probably need to update the number in `ThemesPreference` too.

Comment (CrepeTF) [reply app/src/main/res/layout/pref_theme_item.xml:5]: Nice, didn't catch that

File: app/src/main/res/layout/pref_theme_item.xml [hunk 2]
```diff
@@ -15,102 +15,115 @@
...
+            <com.google.android.material.card.MaterialCardView
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_marginStart="4dp"
                 android:layout_marginTop="4dp"
-                android:background="@drawable/rounded_rectangle"
                 app:layout_constraintStart_toStartOf="@+id/cover_container"
-                app:layout_constraintTop_toTopOf="@+id/cover_container">
+                app:layout_constraintTop_toTopOf="@+id/cover_container"
+                app:cardCornerRadius="6dp">
```
Comment (arkon) [inline app/src/main/res/layout/pref_theme_item.xml:68]: I'm not sure how I feel about the radii for the cover/badges. The previous implementation actually represented the real UI.

Comment (CrepeTF) [reply app/src/main/res/layout/pref_theme_item.xml:68]: but generally when things become smaller, it may be better to exaggerate the shape

**Pull request merged**